app/controllers/entries_controller.rb

```diff
@@ -1,12 +1,15 @@
-class EntryController < ApplicationController
+class EntriesController < ApplicationController
   #before_action :find_entry, only: [:show, :update, :edit, :destroy]
+  #before_action :find_list, only: [:show, :update, :edit, :destroy]
 
   def index
     @entries = Entry.all.order("created_at DESC")
+    @lists = List.all.order("created_at DESC")
```

Comment:
Doesn't look like this var is used for anything in your lists/index.html.erb

Reply:
Oh yeah, I thought I needed to pass through the lists variable from index initially. I removed it now